src/components/Molecules/CardItem/CardItem.css

```diff
@@ -84,7 +84,6 @@
 
 .blurb {
   flex: 0 1 100%;
-  margin-top: 1rem;
```

Comment:
I think we need to be more specific on the blurb styling here. @patrickocoffeyo 

.blurb {
    flex: 0 1 100%;
   margin-top: 1rem;
}

div.blurb {
   margin-top: 0;
}

Reply:
I'll commit this up :)